tests/integration/controller/kyma/kyma_test.go

```diff
@@ -182,41 +193,47 @@ var _ = Describe("Kyma enable one Mandatory Module", Ordered, func() {
 
 	RegisterDefaultLifecycleForKyma(kyma)
 
-	It("should result Kyma in Error state", func() {
-		By("enabling one mandatory Module")
-		kyma.Spec.Modules = append(kyma.Spec.Modules, v1beta2.Module{
-			Name: "mandatory-template-operator",
+	It("should result Kyma in Warning state", func() {
```

Comment:
I am a bit puzzled to be honest why we are updating the test suite for **Mandatory** Modules.

My understanding is that one of the faulty cases that we are addressing with this is for instance that the customer specifies a wrong module name in the KymaCR where the according ModuleTemplate is missing and KymaCR is going to Warning state to indicate that the user needs to update this (I think we have a bug ticket for this somewhere).

For Mandatory Modules my understanding is however that the source of truth (declared state) is the presence of the ModuleTemplate. So if the ModuleTemplate is missing, the manifest should just be de-installed? And since the Mandatory Modules should be transparent to the user, also I would expect that the state of the KymaCR is unaffected, even if there is an error in the Mandatory Module (whether this is good or not is another discussion :))

TL;DR: why do we update the Mandatory Module test suite and not the regular module ones?

Reply:
Lets have a quick sync about this, I think you are missing some points here :D